crates/hir-def/src/attr.rs

```diff
@@ -797,13 +797,16 @@ pub enum AttrInput {
     Literal(SmolStr),
     /// `#[attr(subtree)]`
     TokenTree(tt::Subtree, mbe::TokenMap),
+    /// `#[cfg_attr(cond, inner)]`
+    CfgAttr(CfgExpr, Box<Attr>),
```

Comment:
`CfgAttr` is just a `TokenTree` of a certain form, so this variant shouldn't exist for `AttrInput`